frontend/src/App.tsx

```diff
@@ -8,8 +8,20 @@ import { HostsDataProvider } from "./context/HostContext";
 import { WebSocketProvider } from "./context/WebSocket";
 import NotificationComponent from "./components/Notifications/NotificationComponent";
 import { SnackbarProvider } from "notistack";
+import SockJS from "sockjs-client";
+import { Stomp } from "@stomp/stompjs";
 
 const App: React.FC = () => {
+  const socket = new SockJS("http://localhost:8080/api/v2/ws");
+  const stompClient = Stomp.over(socket);
+  stompClient.connect({}, function (frame: any) {
+    console.log("Connected: " + frame);
+    stompClient.subscribe("/alerts", function (greeting) {
+      console.log("AAAAAAAAA");
```

Comment:
@kraleppa You can do better than this, even after 2 sleepless nights in a row

Reply:
I will kill myself soon, I promise